pennylane/noise/conditionals.py

```diff
@@ -434,12 +566,29 @@ def decorator(f):
     return decorator
 
 
+_MEAS_FUNC_MAP = {
+    qml.expval: qml.measurements.ExpectationMP,
+    qml.var: qml.measurements.VarianceMP,
+    qml.state: qml.measurements.StateMP,
+    qml.density_matrix: qml.measurements.DensityMatrixMP,
+    qml.counts: qml.measurements.CountsMP,
+    qml.sample: qml.measurements.SampleMP,
+    qml.probs: qml.measurements.ProbabilityMP,
+    qml.vn_entropy: qml.measurements.VnEntropyMP,
+    qml.mutual_info: qml.measurements.MutualInfoMP,
+    qml.purity: qml.measurements.PurityMP,
+    qml.classical_shadow: qml.measurements.ClassicalShadowMP,
+    qml.shadow_expval: qml.measurements.ShadowExpvalMP,
+    qml.measure: qml.measurements.MidMeasureMP,
+}
+
+
 def partial_wires(operation, *args, **kwargs):
-    """Builds a partial function based on the given operation with
+    """Builds a partial function based on the given operation or measurement process with
```

Comment:
As the name indicates, the `partial_wires` is only intended to work with things that use a `wires` parameters as supporting everything is impossible and I leave the onus on to the user to rather construct their own noise function as a callable based on the documentation. That being said, I might try to address it setting control to `None`.

Reply:
Following does kinda work, but I still would want users to use this more on atomic operations rather than templates. 
![image](https://github.com/user-attachments/assets/dc833169-d3dc-4fd7-adc8-5c7d94a70464)

Reply:
This should work now!